device/isis/routeradvanced.yaml

```diff
@@ -0,0 +1,71 @@
+components:
+  schemas:
+    Isis.Advanced:
+      description: |-
+        Contains ISIS router advanced properties.
+      type: object
+      properties:
+        enable_hello_padding:
+          description: >-
+            It enables padding of Hello message to MTU size.
+          type: boolean
+          default: true
+        max_area_addresses:
+          description: >-
+            The Number of Area Addresses permitted, with a valid range from 0 to 254. 
+            A zero indicates a maximum of 3 addresses. 
+          type: integer
+          minimum: 0
+          maximum: 254
+          default: 3
+        area_addresses:
+          description: >-
+            Its combination of the ISP and HO-DSP.Usually all nodes within an area have 
+            the same area address e.g. "490001". 
+          type: array
+          items:
+            type: string
+            format: hex
+        lsp_refresh_rate:
+          description: >-
+            The rate at which LSPs are re-sent in seconds.
+          type: integer
+          minimum: 1
+          maximum: 294967295
```

Comment:
What number is this?  Where does it come from?

Reply:
This has been copied from IxN data xml. I shall correct this.

Reply:
Its value should be upto maxint. Hence maximum value is unneccessary.